### ISSUE TITLE
Suggestion (add easter egg map command)

### ISSUE THREAD

**Issue by GCPins** [oliver4888/Impostor-LobbyCommands#1]
Just as the title suggests, allow the "/map" command to include an option to toggle the map to 'Dleks', the reversed version of the Skeld map normally found when hosting lobbies on April 1st.

**Comment by oliver4888** [comment]
I don't believe this is currently possible in the Impostor API, however there is an open issue to do that https://github.com/Impostor/Impostor/issues/147. If it gets added to the API then I will definitely add it to this plugin.
